install.sh

```diff
@@ -85,10 +85,10 @@ else
   echo "[INFO] No errors detected"
 fi
 
-if [[ $collections =~ desktop ]]; then
+rm -f /etc/nologin
+if [[ $collections =~ desktop && ! $collections =~ skip-shutdown ]]; then
```

Comment:
Shutting down and then restarting was wasting time in github actions and also causing some failures which I couldn't explain so I've added this crude method to skip the shutdown.